packages/web-components/src/tablist/tablist.template.ts

```diff
@@ -0,0 +1,8 @@
+import { html, slotted } from '@microsoft/fast-element';
+import { Tablist } from './tablist.js';
+
+export const template = html<Tablist>`
+  <template role="tablist">
+    <slot name="tab" ${slotted('tabs')}></slot>
```

Comment:
I think the old tabs had a start and end slot. Might no longer be required but pretty sure it had those slots.

Reply:
It did have start/end slots. Happy to add them back, but my thinking was that because this is now more composable you wouldn't need them in the tab control itself.

```html
<div class="flex">
  <button>my before content</button>
  <fluent-tablist>...</fluent-tablist>
  <button>my after content</button>
</div>
```

Reply:
Cool I actually like that way better. carry on.